SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -81,6 +105,11 @@ struct ConfigureConnection
 		Params.network.kcp.security_type = WORKER_NETWORK_SECURITY_TYPE_INSECURE;
 		Params.network.tcp.security_type = WORKER_NETWORK_SECURITY_TYPE_INSECURE;
 
+		// Unreal GDK version
+		UnrealGDKVersionPair.name = "gdk_version";
+		UnrealGDKVersionPair.version = TCHAR_TO_ANSI(*GDKVersionLoader::GetGDKVersion());
```

Comment:
Definitely ANSI and not UTF8?

Reply:
I think we should be good with ANSI no? I can't imagine worker would use UTF8 in a c library?

Reply:
I'm basing this off the WorkerType and other vars that we pass to the WorkerSDK being `FTCHARToUTF8`. What do the logs look like from the runtime when you have this param set, print alright?

Reply:
Yeah you are right. Asked worker and they want it UTF8.